<?php
namespace classes\exceptions\user;

use Throwable;

class UseGetValueException extends \Exception
{
    public function __construct($message = "", $code = 0, Throwable $previous = null)
    {
        parent::__construct($message, $code, $previous);
    }
}